docs/en/FACL.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: global
+title: Access Control List
+nickname: Access Control List
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+This document describes the Access Control List feature related to file and directory permissions in Alluxio. 
+
+1. [ACL Permission Model](#ACL Permission Model): Alluxio implements an ACL model similar to those found in Linux and HDFS. This section describes some background on what this permission model looks like. 
+1. [Command Line Interface](#Command Line Interface): The users can interact with Alluxio's ACL model by using the command line interface tools such as `setfacl` and `getfacl`. 
+
+## ACL Permission Model
+Alluxio already implements a file and directory permission model similar to traditional Unix permission bits. For each directory and file, there are three types of permissions: read, write and execute. These permissions can be assigned to three different user clases: owner, group and other. 
+
+The permission bits model is sufficient for most cases. However, it is not flexible enough to express more complicated security policies found in today's computing environment. Access Control List (ACL) is a feature which allows users to mnanage any user or group's rights to any file system objects.
```

Comment:
If the literature talks about Access Control List in the singular we should stay consistent with them, but as an ACL outsider it sounds like it should be plural, e.g. "Access Control Lists allow users to manage"